pkgs/development/python-modules/ipython/5.nix

```diff
@@ -38,6 +39,14 @@ buildPythonPackage rec {
     substituteInPlace setup.py --replace "'gnureadline'" " "
   '';
 
+  patches = [
+    (fetchpatch {
```

Comment:
Why? Please add a comment.

Reply:
Because sage applied that patch and relies on it. As I said in the PR description, I'm guessing/hoping there will be a new ipython release before the final sage 8.3. So that patch will likely never make it into nixpkgs master.

Reply:
Also see https://groups.google.com/forum/#!topic/sage-packaging/rUYd5M5tPTM